Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM eclipse-temurin:17-jre@sha256:0adcf8486107fbd706de4b4fdde64c2d2e3ead4c689b2fb7ae4947010e1f00b4
 
-RUN addgroup spring
-RUN useradd -g spring spring
+RUN addgroup spring&& useradd -g spring spring
```

Comment:
&& without white spaces before is a typo or correct?

Reply:
it's a typo